service/worker/pernamespaceworker_test.go

```diff
@@ -192,8 +192,22 @@ func (s *perNsWorkerManagerSuite) TestEnabled() {
 	cli1.EXPECT().Close()
 }
 
+/*
+		Given machine has ownership of the namespace
+		When name space change reported
+		Then worker should be started with configuration proportional to administratively set configuration ensuring
+	    fair distribution across all the machines owning the namespace
+*/
 func (s *perNsWorkerManagerSuite) TestMultiplicity() {
-	ns := testns("ns3", enumspb.NAMESPACE_STATE_REGISTERED) // three workers
+	machinesOwningNamespace := []membership.HostInfo{
+		membership.NewHostInfoFromAddress("other-1"),
+		membership.NewHostInfoFromAddress("other-2"),
+		membership.NewHostInfoFromAddress("self"),
+	}
+	desiredWorkersNumber := 6
```

Comment:
does this have to match the `6` on line 88? if so, either use a single shared constant, or add a comment explaining that they're linked